README.md

```diff
@@ -1,2 +1,15 @@
 # My-Portfolio
-My very first protfolio
+> My very first protfolio. I reacreate part of a template gave to me by microverse. I used template 4 for my activity.
+> </br>Link: https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10
+
+
+## Built with
+- HTML
+- CSS
+
+## Getting Started
+> Just download or clone my repo and check with an IDE to review my code.
+
+## Author
+
+- GitHub: [@stevers1](https://github.com/Stevers1)
```

Comment:
- [ ] As mentioned by the previous reviewer kindly make sure you use [the Microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)and make your README more professional. I suggest you take a look at [this README](https://github.com/tresorsawasawa/MyPortfolio/blob/develop/README.md) to help you how to fill out all the sections. I know you can do that :muscle: :muscle: